src/main/java/com/bullhornsdk/data/api/StandardBullhornData.java

```diff
@@ -1201,6 +1210,32 @@ protected <C extends CrudResponse, T extends UpdateEntity> C handleUpdateEntity(
         return (C) response;
     }
 
+    /**
+     * Makes the "entity" api call for updating entities, allowing for null bypassing
+     * <p>
+     * HTTP Method: POST
+     *
+     * @param entity
+     * @param nullBypassFields
+     * @return a UpdateResponse
+     */
+    protected <C extends CrudResponse, T extends UpdateEntity> C handleUpdateEntityWithNullBypass(T entity, Set<String> nullBypassFields) {
+        Map<String, String> uriVariables = restUriVariablesFactory.getUriVariablesForEntityUpdate(
+            BullhornEntityInfo.getTypesRestEntityName(entity.getClass()), entity.getId());
+        String url = restUrlFactory.assembleEntityUrlForUpdate();
+
+        CrudResponse response;
+
+        try {
+            String jsonString = restJsonConverter.convertEntityToJsonString(entity);
```

Comment:
Are you missing the 2nd parameter in this `convertEntityToJsonString()`?

Reply:
Totally correct. Adding